src/dev/run_precommit_hook.js

```diff
@@ -6,13 +6,17 @@
  * Side Public License, v 1.
  */
 
+import SimpleGit from 'simple-git/promise';
+
+import { REPO_ROOT } from '@kbn/utils';
 import { run, combineErrors, createFlagError } from '@kbn/dev-utils';
 import * as Eslint from './eslint';
 import * as Stylelint from './stylelint';
 import { getFilesForCommit, checkFileCasing } from './precommit_hook';
 
 run(
   async ({ log, flags }) => {
+    const simpleGit = new SimpleGit(REPO_ROOT);
```

Comment:
Nit: I think we should initialize this in the `if (flags.fix) {` block

Reply:
Nice catch 👍🏼 Unnecessary to create an instance 😅